linked-events.swagger.yaml

```diff
@@ -606,12 +606,28 @@ paths:
         <p>Example:</p>
         <pre><code>organization/?parent=ahjo:100</code></pre>
         <p><a href="?parent=ahjo:100" title="json">See the result</a></p>
+
+        <h4 id="organization-dissolved">Dissolved organizations</h4>
+        <p>To filter organization that are either dissolved use the parameter <code>dissolved</code>.
+            <br/>The filter filters organizations that are either dissolved or not based on the
+            <code>dissolution_date</code>.</p>
+
+        <p>Get only dissolved organizations</p>
+        <p>Example:</p>
+        <pre><code>organization/?dissolved=true</code></pre>
+        <p><a href="?dissolved=true" title="json">See the result</a></p>
+
+        <p>Get organizations not dissolved</p>
+        <p>Example:</p>
+        <pre><code>organization/?dissolved=false</code></pre>
+        <p><a href="?dissolved=false" title="json">See the result</a></p>
       operationId: organization_list
       parameters:
         - $ref: "#/components/parameters/page_param"
         - $ref: "#/components/parameters/pagesize_param"
         - $ref: "#/components/parameters/organization_child_param"
         - $ref: "#/components/parameters/organization_parent_param"
+        - $ref: "#/components/parameters/organization_dissolved_param"
```

Comment:
definition missing from `parameters`  : https://github.com/City-of-Helsinki/api-linked-events/blob/ac23c7ae6d9fa841890e9c4d9931c3e42c7719f4/linked-events.swagger.yaml#L2451C6-L2451C6